app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -166,6 +167,7 @@ private void setupVideoButton(final String videoURI) {
                 i.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
                 try {
                     getContext().startActivity(i);
+                    FormEntryActivity.mFormController.getFormAnalyticsHelper().recordVideoPlaybackStart(videoFile);
```

Comment:
yeah, they are only using external videos for now. Do you think it's still a good idea to log inline videos as well as the same event type ?

Reply:
I think in general it would be useful to add the events for inline, but not CAS's priority.

Maybe a good thing for @ShivamPokhriyal to take a look at?

Reply:
yes! Added a ticket for Shivam [here](https://dimagi-dev.atlassian.net/browse/MOB-142)